createInvoice/src/main/java/com/cloudelements/demo/JSONGetController.java

```diff
@@ -0,0 +1,89 @@
+package com.cloudelements.demo;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.charset.Charset;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+public class JSONGetController {
+
+	@RequestMapping (value = "/getJSON", produces = "application/json")
+	public String getJSON () throws MalformedURLException, IOException {
```

Comment:
For readability, it might be nice to put this JSON in its own file.

Reply:
removed file, it is not part of this project